pkg/network/node/pod.go

```diff
@@ -33,9 +33,10 @@ import (
 )
 
 const (
-	podInterfaceName = "eth0"
-	hostLocalDataDir = "/var/lib/cni/networks"
-	cniPluginsBinDir = "/host/opt/cni/bin"
+	podInterfaceName               = "eth0"
+	hostLocalDataDir               = "/var/lib/cni/networks"
+	cniPluginsBinDir               = "/host/opt/cni/bin"
```

Comment:
is cniPluginsBinDir even used anymore? I don't think so.

Reply:
Good eye!